digest-tiger/lib/digest/tiger/version.rb

```diff
@@ -1,5 +1,5 @@
 module Digest
-  class Tiger
+  class Tiger < Digest::Base
```

Comment:
:+1: 

By the way, what do you think about checking Ruby version in runtime to keep it compatible with previous Ruby versions as well? E.g.
```ruby
if Gem::Version.new(RUBY_VERSION) >= Gem::Version.new('2.3.0')
  class Tiger < Digest::Base
    ...
  end
else
  class Tiger
    ...
  end
end
```